app/presenters/activity_presenter.rb

```diff
@@ -0,0 +1,30 @@
+class ActivityPresenter < SimpleDelegator
+  include ActionView::Helpers::SanitizeHelper
+
+  def self.activities(*activities)
+    activities.flatten.map { |activity| ActivityPresenter.new(activity) }
+  end
```

Comment:
I'd make it take a user/room/jam instead, like so
```
 def self.activities(owner)
    owner.activities.map { |activity| new(activity) }
  end
```

Reply:
I like this. Would have to change https://github.com/tomekr/jamroulette/pull/27#discussion_r399906543. Possibly `<% if @current_user&.activities&.any? %>` and then pass `owner: @current_user` to the partial?